oauth2client/pkce.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
... and it's not even as we've done; it was the jokers before us! :-P

(Apologies for passing the buck.)